sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_server_events_v1/schema.yaml

```diff
@@ -0,0 +1,379 @@
+fields:
+- name: fxa_server
+  type: STRING
+  mode: NULLABLE
```

Comment:
The ETL query currently selects `fxa_server` as the last column, not the first.  Either the query or this schema should be changed so they match (since `fxa_server` is a clustering column I'd be inclined to change the query to select that column first).